base_tier_validation/models/tier_validation.py

```diff
@@ -246,23 +243,27 @@ def _add_comment(self, validate_reject):
             "context": {
                 "default_res_id": self.id,
                 "default_res_model": self._name,
-                "default_definition_ids": definition_ids.ids,
+                "default_review_ids": reviews.ids,
                 "default_validate_reject": validate_reject,
             },
         }
 
     def validate_tier(self):
         self.ensure_one()
+        sequences = self._check_approve_sequence(self.env.user)
```

Comment:
maybe this is no longer "_check_approve_sequence" but something like "_get_sequences_to_approve", what do you think?

Reply:
OK, I agree. Fixed.